### PR TITLE
 Add Project and About Myself Sections

### PR DESCRIPTION
**Pull request by migo1**
###  What I worked on
- learned to work with CSS grids
- Add images to a page

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -113,3 +113,146 @@ header {
   list-style-type: none;
   padding-bottom: 5rem;
 }
+
+.projects {
+  margin: 20% 6% 10% 6%;
+}
```
Comment (braincee) [inline style.css]: - [x] Please I suggest adding CSS `display: grid` to the `projects` class per the project requirements. 😄

File: style.css [hunk 2]
```diff
@@ -113,3 +113,146 @@ header {
...
+.tech-stack li {
+  list-style: none;
+  padding: 4px 12px;
+  background: #ebebff;
+  color: #6070ff;
+  letter-spacing: 0.03em;
+  border-radius: 8px;
+}
+
+.btn-primary-outline {
```
Comment (braincee) [inline style.css:193]: - [x] [OPTIONAL] I kindly suggest that you add the different button interactions including hover, active (when it's pressed), disabled, and enable (default state) as shown in the Figma template below:


Screenshot (248)

the button on the pressed state has the background:  #2230D2; and on hover has the background: #6070FF;

**Example:**

**For pressed state**
```
button:active {
   background-color: #2230D2;
}
```

**For hover state**

```
button:hover {
    background-color: #6070FF;
}
```

File: style.css [hunk 3]
```diff
@@ -113,3 +113,146 @@ header {
...
+
+.projects {
+  margin: 20% 6% 10% 6%;
+}
+
+.card {
+  background-color: #fff;
+  border-radius: 16px;
+  padding: 16px;
+}
```
Comment (braincee) [inline style.css]: - [x] Please I highly recommend using CSS flexbox for displaying individual `cards` per the [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md). 😄

File: index.html [hunk 4]
```diff
@@ -59,5 +59,217 @@ <h3 class="connect">LET’S CONNECT</h3>
...
+          </ul>
+          <button class="btn-primary-outline">See project</button>
+        </div>
+      </div>
+    </section>
+    <!-- </div> -->
+    <!-- About -->
+    <section class="about">
+      <div class="about-container">
+        <h2 class="about-title">About Myself</h2>
```
Comment (braincee) [inline index.html:169]: - [x] Please I highly suggest increasing the pixels `px` to the "About Myself" to fit the required [figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=37-2238&t=HQA60gDOdof1M29t-0). 😄 

_**REQUIRED FIGMA DESIGN**_

Screenshot (252)

_**YOUR DESIGN**_


Screenshot (250)

**Review by braincee — CHANGES_REQUESTED** [review]
Hello @migo1 👋 , I am @braincee your Code Reviewer

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close

HIGHLIGHTS
------

 - GitHub flow rightfully used ✅
 - Meaningful title and summary for PR ✅
 - Professional README.md file ✅
 - No linter errors ✅

REQUIRED CHANGES ♻️
------
**_Please check the comments under the review._**

OPTIONAL CHANGES
--------

- [ ] [OPTIONAL] Please I highly suggest capitalizing your commit messages to make your repository look professional. Example `Add...` instead of `add..`. Please [check this link](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) for more information on how to document your repository professionally. 😄 

**_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._**

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @braincee in your question so I can receive the notification.

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._

----

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (migo1) [reply style.css]: added the display property

File: style.css (hunk 2 above)
Comment (migo1) [reply style.css:193]: modified the button states

File: style.css (hunk 3 above)
Comment (migo1) [reply style.css]: added flex column

File: index.html (hunk 4 above)
Comment (migo1) [reply index.html:169]: increased the px

Comment (migo1) [reply index.html:169]: @braincee thanks for the review,

**Review by TadesseDev — APPROVED** [review]
### Hi @migo1 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
giphy (1)



## Highlights
 - [x] Good job making the changes requested by the previous code reviewer. ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**